src/templates/organization.js

```diff
@@ -272,35 +261,25 @@ export const query = graphql`
         LinkedIn_Profiles {
           data {
             Logo {
-              localFiles {
-                childImageSharp {
-                  fixed(
-                    width: 64
-                    height: 64
-                    fit: CONTAIN
-                    background: "white"
-                  ) {
-                    ...GatsbyImageSharpFixed
-                  }
-                }
-              }
+              ...OrganizationCardLogo
+            }
+          }
+        }
+        Crunchbase_ODM {
+          data {
+            Logo {
+              ...OrganizationCardLogo
             }
           }
         }
         Categories {
           id
           data {
             Name
-            Cover {
-              ...OrganizationCardPhoto
-            }
             Parent {
               id
               data {
                 Name
-                Cover {
-                  ...OrganizationCardPhoto
-                }
```

Comment:
this is useful as a fallback if the org doesn't have a logo

Reply:
I'm happy with that on the list view, but on this page it's not adding value.